app/controllers/events/view/sessions/create.js

```diff
@@ -3,13 +3,13 @@ export default Controller.extend({
   actions: {
     save() {
```

Comment:
That's a lot of then & catch. Use `async` function and make this linear and more-readable.